docs/src/snap/reference/annotations.md

```diff
@@ -1,34 +1,187 @@
 # Annotations
 
 This page outlines the annotations that can be configured during cluster
```

Comment:
do we want to link to somewhere that explains what annotations are in this context? Like https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/ maybe?

Reply:
Thats a good idea. I'll add that now.